pie/bot.py

```diff
@@ -58,6 +72,11 @@ async def update_app_info(self):
         else:
             self.owner_ids = {app.owner.id}
 
+    async def close(self, signal: str = None) -> None:
+        message = f"Signal{signal} received. " if signal else ""
```

Comment:
No space is here required? I would use something like 

`message = f"Signal '{signal}' received. " if signal else "Unknown"`

Reply:
The `close()` method is not used only for the signal handling, but also for shutting down the bot from !strawberry restart / shutdown commands. In those cases, the message can't contain signal. However, I might have an idea how to handle this better way.

Reply:
Please check latest commit. It should be addressed

Reply:
LGTM